src/patterns/utilities/space.hbs

```diff
@@ -0,0 +1,79 @@
+---
+name: Space
+previewClass: u-spaceItems1
+notes: |
+  Utilities for applying whitespace adjustments (`margin` or `padding`) to
+  elements.
+
+  + `u-spaceX` applies `margin` all around.
+  + `u-spaceEndsX` applies `margin` to top and bottom.
+  + `u-spaceSidesX` applies `margin` to left and right.
+  + `u-spaceTopX` applies `margin` to top.
+  + `u-spaceBottomX` applies `margin` to bottom.
+  + `u-spaceLeftX` applies `margin` to left.
+  + `u-spaceRightX` applies `margin` to right.
+  + `u-spaceItemsX` applies vertical `margin` between adjacent items.
+  + `u-padX` applies `padding` all around.
+  + `u-padEndsX` applies `padding` to top and bottom.
+  + `u-padSidesX` applies `padding` to left and right.
+  + `u-padTopX` applies `padding` to top.
+  + `u-padBottomX` applies `padding` to bottom.
+  + `u-padLeftX` applies `padding` to left.
+  + `u-padRightX` applies `padding` to right.
+  + `u-pullSidesX` applies negative `margin` to left and right.
+  + `u-pullLeftX` applies negative `margin` to left.
+  + `u-pullRightX` applies negative `margin` to right.
```

Comment:
I think the pull utilities might be the only ones of the bunch that don't span the full modular scale, using just the positive values. Might be worth mentioning here.